src/backend/looped/self/super_hero_fly.cpp

```diff
@@ -0,0 +1,404 @@
+#include "backend/looped_command.hpp"
+#include "gta/enums.hpp"
+#include "natives.hpp"
+#include "util/entity.hpp"
+#include "util/math.hpp"
+#include "util/notify.hpp"
+#include "util/ped.hpp"
+#include "util/vehicle.hpp"
+
+#include <ctime>
+
+namespace big
+{
+	class super_hero_fly : looped_command
+	{
+		using looped_command::looped_command;
+
+		bool flying            = false;
+		bool landing           = false;
+		bool launching         = false;
+		bool anims_are_playing = false;
+
+		bool timer = false;
+		std::chrono::system_clock::time_point time_stamp;
+		std::chrono::duration<double> duration;
```

Comment:
Since you are refering to this property later in milliseconds, you should type it as std::chrono::milliseconds, not double.

Reply:
the `duration` variable is closely related with `charge_intensity`, this on its own might be unusual but it does force me to use double for the duration.

Reply:
Then the name of the variable is misleading. Either rename it, or type it as a std::chrono helper time duration.

Reply:
na